ceph.spec.in

```diff
@@ -58,10 +58,14 @@
 %bcond_with cephfs_java
 %bcond_with kafka_endpoint
 %bcond_with libradosstriper
-%ifarch x86_64 aarch64 ppc64le
+%ifarch x86_64 ppc64le
 %bcond_without lttng
 %bcond_without rbd_rwl_cache
 %bcond_without rbd_ssd_cache
+%elifarch aarch64
+%bcond_without lttng
+%bcond_with rbd_rwl_cache
+%bcond_without rbd_ssd_cache
```

Comment:
No problem with disabling the `rwl` mode by default on aarch64, but I don't understand why the `ssd` mode gets special treatment in the spec file.  It doesn't depend on PMDK or any other special libraries, so I think should be enabled by default regardless of the architecture and distribution.

Reply:
@idryomov #42006 is created to address this issue.

Reply:
i will rebase this PR once #42006 gets merged (if it is approved).

Reply:
I approved #42006.  For this PR, I think we should move `rbd_rwl_cache` manipulation to the top level because the architectures that it is supported on (x86_64 ppc64le) are the same for Fedora/RHEL and SUSE.

Reply:
@idryomov agreed.

@kevinzs2048 could you rebase on top of my change and address Ilya's comment?

Reply:
Thanks for the review. @idryomov @tchaikov 
Done. 
Disable ppc64 as well as PMDK is only official support on x86